Week1/Js-exercises/3-log-the-number.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+let numberX;
+console.log("I think x is undefined");
+console.log(x);
+x = 84;
+console.log("now x should be 26");
+console.log(x);
```

Comment:
Wait, what? If you do `x = 84;`, why will it be 26? 🤔

Reply:
Fixed , x will be 84 .